docker-compose.yml

```diff
@@ -23,33 +26,34 @@ services:
 
   maildev:
     image: djfarrelly/maildev
-    env_file: .env
+    env_file: ${PROJECT_LOCATION}/docker/local/.env
     depends_on:
       - php
     ports:
-      - "1080:80"
+      - 1080:80
 
   mysql:
     image: mariadb:latest
-    env_file: .env
+    env_file: ${PROJECT_LOCATION}/docker/local/.env
     ports:
-      - "3306:3306"
+      - 3306:3306
     volumes:
       - mysql:/var/lib/mysql
     tty: true
 
   nginx:
-    build: nginx
-    env_file: .env
+    build:
+      context: ./nginx
+    env_file: ${PROJECT_LOCATION}/docker/local/.env
     ports:
-      - "80:80"
+      - 443:443
     volumes:
       # Projects files
-      - ${WEBSITES}:/var/www/html:rw,delegated
+      - code_sync:/var/www/html:nocopy
       # Websites configuration
-      - ./nginx/site-available:/etc/nginx/conf.d/:ro
+      - ${PROJECT_LOCATION}/docker/local/nginx.conf:/etc/nginx/conf.d/custom.conf:ro
       # Logs
-      - ./nginx/log:/var/log/nginx
+      - ${PROJECT_LOCATION}/docker/log/nginx:/var/log/nginx
```

Comment:
You won't be able to extract Nginx logs by using the default image as everything is redirected to the standard output.

```
ls -l /var/log/nginx/
total 0
lrwxrwxrwx    1 root     root            11 Sep 24 23:33 access.log -> /dev/stdout
lrwxrwxrwx    1 root     root            11 Sep 24 23:33 error.log -> /dev/stderr
```

Reply:
I don't know how, but logs are correctly extract on my project

<img width="1437" alt="Capture d’écran 2019-10-15 à 12 24 26" src="https://user-images.githubusercontent.com/5331654/66823748-2987e880-ef47-11e9-8c6b-a0c7b1fc462b.png">

Reply:
Indeed, it seems that now the volume declaration bypasses the redirect in the filesystem...

But even if the extract works, keep in mind that you won't be able anymore to display logs through `docker-compose logs nginx`. I don't know if it's important for you. 😉